nri/css/style.css

```
@@ -0,0 +1,652 @@
+html{
+  font-size: 62.5%;
+  font-family: sans-serif;
+  line-height: 1.5;
+}
+
+body{
+  padding:0px;
+  text-align:center;
+}
+
+/*header領域*/
+
+.nri-header{
+  background-color: #fff;
+  height: 60px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 1000;
+}
+
+.header-logo{
+  margin: 0 25px 0 25px;
+  width: 58px;
+  height: auto;
+  transition: all .4s;
+  box-sizing: border-box;
+}
+
+.main-menu {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.header-main-menu{
+  box-sizing: border-box;
+}
+
+.global-menu{
+  display: flex;
+  font-size: 26px;
+  right: 0;
+  padding: 0;
+  margin-right: 35px;
+  list-style: none;
+}
+
+header ul{
```

```suggestion
.nri-header ul {
```

特に一番左にはタグ名を直接書かないほうがベターです。

せっかく`<header class="nri-header">`このようにマークアップしているので
（タグに直接スタイルをあてると、cssの優先順位が高いため変更しにくいから）